style.css

```diff
@@ -0,0 +1,85 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  height: 100%;
+}
+
+body {
+  background-color: white;
+  color: #b13f6a;
+  text-align: center;
+  font-family: "Jost", sans-serif;
+  font-size: 20px;
+}
+
+.filter-bar {
+  text-align: left;
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+}
+
+.gallery-wrapper {
+  background-color: #b13f6a;
+  color: #ffb000;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  padding: 20px;
+  box-sizing: border-box;
+}
+
+.card-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #ffb000;
+  color: #b13f6a;
+  width: calc(33.33% - 40px); /* 33.33% for 3 columns, subtracting margins */
+  margin: 20px;
+  padding: 20px;
+  box-sizing: border-box;
+  border-radius: 20px;
+  border: solid 2px rgb(205, 117, 30);
+}
+
+button {
```

Comment:
to improve the UI on mobile screens, reduce the size of paddings/margins for this element, it takes up too much space. Font size can be reduces too in the parent elements 👍 
<img width="378" alt="Screenshot 2024-03-04 at 09 27 29" src="https://github.com/Technigo/project-library/assets/33493978/00aa4fd2-a316-4ef6-b197-18d71b3b52f6">